resources/language/en_us.ini

```diff
@@ -0,0 +1,1565 @@
+; this message gets displayed when the provided player has no plots claimed
+; {%3} = the provided players name
+visit.oneArgument.player.noPlots = "§6{%0} §cdoesn't own any plots."
+; this message gets displayed when there was an error during teleporting the command executor to the plot
+; {%0} = the world name of the plot
+; {%1} = the x coordinate of the plot
+; {%2} = the z coordinate of the plot
+; {%3} = the provided players name
+visit.oneArgument.player.teleportError = "§cThere was an error while teleporting you to plot (§6{%1}§c;§6{%2}§c)§8: §7{%0}"
+; this message gets displayed when the player was successfully teleported to the plot
+; {%0} = the world name of the plot
+; {%1} = the x coordinate of the plot
+; {%2} = the z coordinate of the plot
+; {%3} = the provided players name
+visit.oneArgument.player.success = "§7You were teleported to plot (§6{%1}§c;§6{%2}§c)§8: §7{%0}"
+; this message gets displayed when there couldn't be found a plot with the provided plot alias
+; {%0} = the provided plot alias
+visit.oneArgument.alias.noPlot = "§cThere is no plot with alias §6{%0}§c."
+; this message gets displayed when there was an error during teleporting the command executor to the plot
+; {%0} = the world name of the plot
+; {%1} = the x coordinate of the plot
+; {%2} = the z coordinate of the plot
+; {%3} = the provided plot alias
+visit.oneArgument.alias.teleportError = "§cThere was an error while teleporting you to plot (§6{%2}§c;§6{%3}§c)§8: §7{%0}"
+; this message gets displayed when the command executor was successfully teleported to the plot
+; {%0} = the world name of the plot
+; {%1} = the x coordinate of the plot
+; {%2} = the z coordinate of the plot
+; {%3} = the provided plot alias
+visit.oneArgument.alias.success = "§7You were teleported to plot (§6{%2}§c;§6{%3}§c)§8: §7{%0}"
+; this message gets displayed when the provided player name neither doesn't match with an online player or an in the database existing player
+; {%0} = the provided player name
+visit.twoArguments.playerNotFound = "§cPlayer §6{%0} §ccould not be found."
+; this message gets displayed when the provided player has no plots claimed
+; {%0} = the provided players name
+visit.twoArguments.noPlots = "§6{%0} §cdoesn't own any plots."
+; this message gets displayed when the provided player has no plot with the provided number
+; {%0} = the provided players name
+; {%1} = the provided plot number
+visit.twoArguments.noPlot = "§6{%0} §cdoesn't own a plot with number §6{%1}§c."
+; this message gets displayed when there was an error during teleporting the command executor to the plot
+; {%0} = the provided plot number
+; {%1} = the world name of the plot
+; {%2} = the x coordinate of the plot
+; {%3} = the z coordinate of the plot
+; {%4} = the provided players name
+visit.twoArguments.teleportError = "§cThere was an error while teleporting you to plot (§6{%2}§c;§6{%3}§c)§8: §7{%0}"
```

Comment:
Maybe `§cThere was an error while teleporting you to plot §6{%0} §c(§6{%2}§c;§6{%3}§c) of §6{%4}§c.`